includes/class-wc-order.php

```diff
@@ -420,7 +420,7 @@ public function set_status( $new_status, $note = '', $manual_update = false ) {
 	public function update_status( $new_status, $note = '', $manual = false ) {
 		try {
 			if ( ! $this->get_id() ) {
-				throw new Exception( 'Missing ID' );
+				throw new Exception();
```

Comment:
You can just return false here. No reason to have an `Exception`, since is not sending any message.